glean-core/android/src/main/java/mozilla/telemetry/glean/Dispatchers.kt

```diff
@@ -29,9 +29,9 @@ internal object Dispatchers {
         // Use a [ConcurrentLinkedQueue] to take advantage of it's thread safety and no locking
         internal val taskQueue: ConcurrentLinkedQueue<suspend CoroutineScope.() -> Unit> = ConcurrentLinkedQueue()
 
-        private val logger = Logger("glean/Dispatchers")
-
         companion object {
+            private const val LOG_TAG = "Dispatchers"
```

Comment:
Why are we changing the tag here? The maximum tag size is 23 characters, so we should fit.